### DISCUSSION TITLE
whice version of xformes should i use?version 0.0.17.dev484 cant loading.

### DISCUSSION THREAD

**Discussion by sumic** [d8ahazard/sd_dreambooth_extension#1115] [Q&A]
`
Python 3.10.6 | packaged by conda-forge | (main, Oct 24 2022, 16:02:16) [MSC v.1916 64 bit (AMD64)]
Commit hash: a9fed7c364061ae6efb37f797b6b522cb3cf7aa2
Installing requirements for Web UI

Initializing Dreambooth
If submitting an issue on github, please provide the below text for debugging purposes:

Python revision: 3.10.6 | packaged by conda-forge | (main, Oct 24 2022, 16:02:16) [MSC v.1916 64 bit (AMD64)]
Dreambooth revision: da2e40415f1cb63cc4de46d6dc97eb8676c6e30c
SD-WebUI revision: a9fed7c364061ae6efb37f797b6b522cb3cf7aa2

Successfully installed accelerate-0.17.1
Successfully installed requests-2.28.2
Successfully installed fastapi-0.90.1 starlette-0.23.1
Successfully installed gitpython-3.1.31
Successfully installed transformers-4.27.2

[+] torch version 1.13.1+cu117 installed.
[+] torchvision version 0.14.1+cu117 installed.
[+] xformers version 0.0.17.dev484 installed.
[+] accelerate version 0.17.1 installed.
[+] diffusers version 0.14.0 installed.
[+] transformers version 4.27.2 installed.
[+] bitsandbytes version 0.35.4 installed.

Launching Web UI with arguments: --xformers
WARNING[XFORMERS]: xFormers can't load C++/CUDA extensions. xFormers was built for:
    PyTorch 2.0.0+cu118 with CUDA 1108 (you have 1.13.1+cu117)
    Python  3.10.10 (you have 3.10.6)
  Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
  Memory-efficient attention, SwiGLU, sparse and more won't be available.
  Set XFORMERS_MORE_DETAILS=1 for more details
`
 xFormers(0.0.17) was built for:
    PyTorch 2.0.0+cu118 with CUDA 1108 (you have 1.13.1+cu117)
    Python  3.10.10 (you have 3.10.6)
how can i use it ?
if i use automatic1111 install xformers, the xformers version is 0.0.16,cant start dreambooth.
which version should i use?
thx a lot.

**Comment by sogngenWang** [comment]
use below command seems ok : 
 pip install git+https://github.com/facebookresearch/xformers@v0.0.17rc482

**Reply by sumic** [reply]
> use below command seems ok : pip install git+https://github.com/facebookresearch/xformers@v0.0.17rc482

WARNING[XFORMERS]: xFormers can't load C++/CUDA extensions. xFormers was built for:
    PyTorch 2.0.0+cu118 with CUDA 1108 (you have 1.13.1+cu117)
    Python  3.10.10 (you have 3.10.6)
  Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
  Memory-efficient attention, SwiGLU, sparse and more won't be available.
  Set XFORMERS_MORE_DETAILS=1 for more details


still not work for me.

**Reply by sumic** [reply]
i think if i update the python and PyTorch  version the SD maybe cant work.
